service/controller/legacy/drainer.go

```diff
@@ -11,14 +11,16 @@ import (
 	"github.com/giantswarm/operatorkit/controller"
 	"github.com/giantswarm/operatorkit/informer"
 	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 
 	awsclient "github.com/giantswarm/aws-operator/client/aws"
-	"github.com/giantswarm/aws-operator/service/controller/legacy/v25"
-	"github.com/giantswarm/aws-operator/service/controller/legacy/v26"
-	"github.com/giantswarm/aws-operator/service/controller/legacy/v27"
-	"github.com/giantswarm/aws-operator/service/controller/legacy/v28"
-	"github.com/giantswarm/aws-operator/service/controller/legacy/v29"
+	"github.com/giantswarm/aws-operator/service/controller/key"
+	v25 "github.com/giantswarm/aws-operator/service/controller/legacy/v25"
+	v26 "github.com/giantswarm/aws-operator/service/controller/legacy/v26"
+	v27 "github.com/giantswarm/aws-operator/service/controller/legacy/v27"
+	v28 "github.com/giantswarm/aws-operator/service/controller/legacy/v28"
+	v29 "github.com/giantswarm/aws-operator/service/controller/legacy/v29"
```

Comment:
Why are we doing this back and forth all the time? I saw this in other PRs and find it super annoying that some people's editors do that and some's don't.

Reply:
I updated `goimports` now and it still does that. What I can do about it?

Reply:
Don't know. My `goimports` doesn't do that.

Reply:
I have this happening as well with new version of goimports.
I think the `vxx`stuff as alias up front is some new standard.

Somehow related to go modules:
https://github.com/golang/go/issues/30051